### PR TITLE
Add logout success message

### PR DESCRIPTION
**Pull request by SableWalnut**
Add a logout success message so users know when they've successfully logged out

### CODE REVIEW

File: wagtail/wagtailadmin/views/account.py [hunk 1]
```diff
@@ -126,6 +125,7 @@ def login(request):
 def logout(request):
     response = auth_views.logout(request, next_page='wagtailadmin_login')
 
+    messages.success(request, 'You have successfully logged out!')
```
Comment (tomdyson) [inline wagtail/wagtailadmin/views/account.py]: Thanks, Liam! The message should be a translatable string, e.g. https://github.com/takeflight/wagtail/blob/logout-message/wagtail/wagtailadmin/views/account.py#L59. I'm not sure we need the `!`.

File: wagtail/wagtailadmin/views/account.py [hunk 2]
```diff
@@ -12,7 +12,6 @@
 from django.utils.translation import ugettext as _
 from django.views.decorators.cache import never_cache
 from django.views.decorators.debug import sensitive_post_parameters
-
```
Comment (dirtycoder) [inline wagtail/wagtailadmin/views/account.py]: This blank line should be kept. :)

**Comment by SableWalnut** [comment]
All fixed up. :)

**Comment by m1kola** [comment]
Merged in ca846a5941e2bdbe82a97b4a99cfea5876172d85. Thanks, Liam!

**Pull request closed without merging**